### ISSUE TITLE
Roll out new code to production?

### ISSUE THREAD

**Issue by dkoslicki** [RTXteam/RTX#161]
@saramsey @edeutsch We've made quite a bit of progress by the looks of http://rtx.ncats.io/devLM/, http://rtx.ncats.io/devED/, and http://rtx.ncats.io/test/. Shall we make plan for rolling this out to http://rtx.ncats.io soon? It would be nice to have things stable prior to the hackathon. Given we only have 1 working day to get this done, I'll mark this high priority (but can be persuaded otherwise given your opinions).

@edeutsch maybe you were waiting on @arnabdotorg's auto-complete code before rolling to production. If so, let me know.

**Comment by edeutsch** [comment]
Hi David, yes, there will be a complete clearing of the feedback database and caching system today, hopefully by noon. If there's any more testing you want to do in test/ now is a good time. There are a couple more bugs I need to squash before the clear the database and roll out. I am not waiting on auto-complete. That will need to go through testing before we consider rolling out. Just a short list of remaining things. I will make an announcement later today when we're refreshed and rolled out and "in production".

**Comment by edeutsch** [comment]
okay, I am about to roll everything out to production. will announce when done.

**Comment by edeutsch** [comment]
The latest code is rolled out to production. This means it is perfect.
heh.
